README.md

```diff
@@ -8,7 +8,7 @@ Code for performing real-time shimming using external MRI shim coils
 
 ## Installation
 
-The first dependency you need is [`dcm2niix`](https://github.com/rordenlab/dcm2niix) >= v1.0.20200331.
+The first dependency you need is [`dcm2niix`](https://github.com/rordenlab/dcm2niix) > v1.0.20201102.
```

Comment:
Are you certain it is only greater than and not greater than or equal to?

Reply:
The original issue never mentioned equal to only the greater than part was mentioned, I'm quoting the issue "we need dcm2niix version to be greater than v1.0.20201102." But do let me know if this should be greater than equal to.

Reply:
Sorry for the confusion, I updated the issue to reflect greater or equal to.

Reply:
Thanks for the clarification I will update this as well then.